modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,64 @@
+= Anypoint Runtime Fabric Overview
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Runtime Fabric is a container service that allows you to run Mule applications, composite APIs and API gateways across any environment with centralized management via Anypoint Platform.
+
+Some of the capabilities of Anypoint Runtime Fabric include:
+
+* Isolation between applications by running a separate Mule runtime server per application.
+* Ability to run multiple versions of Mule runtime server on the same set of resources.
+* Scaling applications across multiple replicas.
+* Automated application fail-over.
+* Application management with Anypoint Runtime Manager.
+
+[NOTE]
+Contact your Customer Success Manager for information on how enable Anypoint Runtime Fabric on your account.
+
+There are two management options for Anypoint Runtime Fabric:
```

Comment:
Per SF, avoid beginning sentences with "There is" or "There are."
```suggestion
Anypoint Runtime Fabric includes two management options:
```
You could add something more, such as "...options, enabling you to further customize your deployment" or something similar.